dist/ramda.js

```diff
@@ -5371,10 +5376,16 @@
      *
      */
     var invert = function invert(obj) {
-        var props = keys(obj), len = props.length, idx = -1, out = {};
+        var props = keys(obj);
+        var len = props.length;
+        var idx = -1;
+        var out = {};
         while (++idx < len) {
-            var key = props[idx], val = obj[key];
-            out[val] = out[val] || [];
+            var key = props[idx];
+            var val = obj[key];
+            if (!has(val, out)) {
+                out[val] = [];
+            }
```

Comment:
Because the one-liner is broken. ;)

The current implementation fails the test case.

Aside: I like to use `&&` and `||` with Boolean operands only. Similarly I ensure each of my predicates returns a Boolean and each of my conditions evaluates to a Boolean. I thus avoid the world of JavaScript truthiness.